tests/v2/actions/scalinginput/replace.go

```diff
@@ -40,13 +37,15 @@ const (
 	clusterLabel                 = "cluster.x-k8s.io/cluster-name"
 )
 
-func MatchNodeToRole(t *testing.T, client *rancher.Client, clusterID string, isEtcd, isControlPlane, isWorker bool) (int, []management.Node) {
+func MatchNodeToRole(client *rancher.Client, clusterID string, isEtcd, isControlPlane, isWorker bool) (int, []management.Node, error) {
```

Comment:
We should probably add a godoc here, I didn't realize we didn't have one for this function.